.github/workflows/ci.yml

```diff
@@ -37,7 +37,29 @@ jobs:
       - name: Build package
         run: |
           python scripts/build/install.py
-
+      
+      - name: Build C++ Code with Coverage
+        run: |
+          mkdir -p build
+          cd build
+          cmake -DCMAKE_CXX_FLAGS="--coverage" ..
```

Comment:
This is not the right way. Nice attempt though.

Reply:
removed this part of the code for now sir ,will work on it after the current issue